contracts/standard/arbitration/MultipleArbitrableTransaction.sol

```diff
@@ -1,18 +1,18 @@
 /**
- * @authors: []
- * @reviewers: []
+ * @authors: [@clesaege, @n1c01a5, @epiqueras, @satello, @ferittuncer, @eburgos, @dannybabbev]
```

Comment:
I didn't write this contract, pretty sure the only authors are "eburgos" and you, @n1c01a5.

Reply:
ok but could we have a contributors parameter?

Reply:
`authors === contributors`

Actually, looks like more of us authored it.

See history: https://github.com/kleros/kleros-interaction/commits/66166fb63ebf71d5aa45412311b693d591921334/contracts/standard/arbitration/MultipleArbitrableTransaction.sol

Reply:
yeah that's reason before I added some authors. Anyway they can do a PR